assets/css/about.css

```diff
@@ -0,0 +1,466 @@
+  font-size: 15px;
+  padding-bottom: 5%;
+  text-decoration: underline;
+}
+
+.conatiner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 20px;
+}
+
+.our-history {
+  border-bottom: 1px solid #c9ccd3;
+  padding-bottom: 50px;
+}
+
+.our-history .title-3 {
+  font-size: 20px;
+  font-family: 'lato', sans-serif;
+  color: #272a31;
+  text-align: center;
+  margin: 20px;
+}
+
+.our-history .title-3::after {
+  content: "";
+  display: block;
+  border-bottom: 1px solid #ec5242;
+  width: 50px;
+  text-align: center;
+  margin: 5px auto;
+}
+
+.our-history .text-2 {
+  font-size: 15px;
+  font-family: 'lato', sans-serif;
+  color: #272a31;
+  text-align: center;
+  margin: 5px;
+  line-height: 30px;
+}
+
+.our-history .history-logo {
+  border: 1px solid #f2f3f5;
+  margin: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.our-history .history-logo img {
+  width: 200px;
+  height: 200px;
+}
+
+.our-history .history-logo .text-1 {
+  color: #272a31;
+  font-family: 'lato', sans-serif;
+  font-size: 18px;
+}
+
+.history-logo .text-3 {
+  color: #272a31;
+  font-family: 'lato', sans-serif;
+  font-size: 15px;
+  margin: 5px;
+  font-style: italic;
+}
+
+.our-experiance {
+  display: flex;
+  flex-direction: column;
+  margin: 10px 0;
+}
+
+.our-experiance .title-3 {
+  color: #ec5242;
+  font-size: 18px;
+  font-family: 'lato', sans-serif;
+  text-align: center;
+}
+
+.our-experiance .text-3 {
+  color: #272a31;
+  font-size: 15px;
+  font-family: 'lato', sans-serif;
+  margin: 5px;
+}
+
+.experiances {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.success {
+  position: relative;
+  margin: 10px;
+  height: 220px;
+  width: 100%;
+}
+
+.experiance-text {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+  opacity: 0;
+  transition: 0.5s ease;
+  background-color: #ff4935;
+}
+
+.success .experiance-img {
+  width: 100%;
+  object-fit: cover;
+  background-color: #ec5242;
+}
+
+.success .experiance-img:hover {
+  background-color: #d63f2e;
+}
+
+.experiance-text .expriance-title {
+  color: white;
+  font-size: 18px;
+  font-family: 'lato', sans-serif;
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+}
+
+.experiance-text .text-3 {
+  color: white;
+  font-size: 15px;
+  font-weight: 400;
+  font-family: 'lato', sans-serif;
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+}
+
+.success .experiance-text {
+  opacity: 0.9;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  background-color: #272a31;
+  padding: 20px;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] Mobile Size:
- Please make sure the footer has white color as stated in the previous review.
-
![n2](https://user-images.githubusercontent.com/63972670/169487359-29466fa4-59cc-496d-875a-a92573e5e650.PNG)

Reply:
Your review is informative and provide important information, I have some question about the mobile layout  footer for about page  in the given design is #272a31; white footer is used for home page only.